tests/rpc/calls/getStorageAt.test.ts

```diff
@@ -0,0 +1,19 @@
+import { getStorageAtHandler } from '../../../src/rpc/calls/getStorageAt'
+import { RPCResponse } from '../../../src/types/types'
+describe('Test get Storage At request testnet', () => {
+  it('Returns storage address', async () => {
+    const request = {
+      jsonrpc: '2.0',
+      method: 'eth_getStorageAt',
+      params: [
+        "0x07cab99e7a284f2486bd49a3e0453f51e904f7a928830f62e22294e77fc45351",
+        "0x0000000000000000000000000000000000000000000000000000000000000001",
+        "latest"
+      ],
+      id: 1,
+    }
+    const result: RPCResponse = <RPCResponse>await getStorageAtHandler(request)
+    expect(typeof result.result).toBe('string')
```

Comment:
Please compare with the exact result.

Reply:
@ermvrs is test case with empty storage slot okay? Actually we can use some non-empty slots, but they changes value frequently, so the test may fail in some point in the future. I haven't figured out a good way to test that. Any suggestion?

Reply:
Oh. We missed one issue to complete this one. I will deploy our test registry and register an address for you. Are you in telegram channel?

Reply:
https://t.me/rosettanet